src/tools/run-make-support/src/diff/mod.rs

```diff
@@ -97,6 +100,15 @@ impl Diff {
             .to_string();
 
         if !output.is_empty() {
+            // If we can bless (meaning we have a file to write into and the `RUSTC_BLESS_TEST`
+            // environment variable set), then we write into the file and return.
```

Comment:
I'll take note to also write this into the support library docs, we probably also should document this in rustc-dev-guide for run-make tests?

Reply:
I suppose so. Didn't know there was a guide. ^^'